### DISCUSSION TITLE
Check this ,it can help you a lot with auto authentication and continued conversation

### DISCUSSION THREAD

**Discussion by Yorddyyorddy** [acheong08/EdgeGPT#512] [Show and tell]
Check this open source repo ,it contained what features needed in your EdgeGPT like conversation  auto authentication, clear markdown.....etc

@acheong08

**Comment by acheong08** [comment]
Which repo?

**Reply by Yorddyyorddy** [reply]
Sorry my damn keyboard erased it 
This one https://github.com/nitanmarcel/sydney-telegram
I don't know how they figured out this all
Demo https://t.me/sydneygpt_bot
